nautobot/ui/package.json

```diff
@@ -30,8 +31,11 @@
     "react-dom": "^18.2.0",
     "react-dotenv": "^0.1.3",
     "react-icons": "^4.7.1",
+    "react-redux": "^8.0.5",
     "react-router-dom": "^6.4.3",
     "react-scripts": "5.0.1",
+    "redux-dynamic-middlewares": "^2.2.0",
```

Comment:
Not used in this phase so should be removed. May not be needed depending on how we generate a full API object.